core/src/test/java/org/datasyslab/geospark/formatMapper/GeoJsonReaderTest.java

```diff
@@ -101,4 +101,11 @@ public void testReadToValidGeometryRDD()
         geojsonRDD = GeoJsonReader.readToGeometryRDD(sc, geoJsonWithInvalidGeometries);
         assertEquals(geojsonRDD.rawSpatialRDD.count(), 3);
     }
+
+    @Test
+    public void setGeoJsonIdHandleTest() throws Exception{
+        String pathd = GeoJsonReaderTest.class.getClassLoader().getResource("idContainsGeoJson.json").getPath();
+        SpatialRDD geojsonRDD = GeoJsonReader.readToGeometryRDD(sc,pathd);
+
```

Comment:
1. This test case doesn't actually test the correctness. Please follow the style of other GeoJSON reader test case. For instance, use `assertEquals(geojsonRDD.rawSpatialRDD.count(), 3);` to check the correctness.
2. Please follow the coding style in other test cases. Please put the test data file path in the `onceExecutedBeforeAll()` function